src/test/java/com/google/ehub/servlets/ItemPageServletTest.java

```diff
@@ -77,9 +78,10 @@
   @Before
   public void init() {
     MockitoAnnotations.initMocks(this);
-    helper.setUp();
     ProfileDatastore profile = new ProfileDatastore();
     profile.addUserProfileToDatastore("Eric", EMAIL, USERNAME, "Hey");
+    helper.setUp();
+
```

Comment:
Is it the same error when it looks like this? 
```suggestion
public void init() {
    MockitoAnnotations.initMocks(this);
    helper.setUp();	
    ProfileDatastore profile = new ProfileDatastore();
    profile.addUserProfileToDatastore("Eric", EMAIL, USERNAME, "Hey");
```